client/src/components/EditAssignmentView.js

```diff
@@ -66,6 +67,21 @@ const EditAssignmentView = () => {
     }
   };
 
+  // Function to delete data from the database or from the context
+  const deleteAttainment = async (attainmentId) => {
+    // If this view is opened from the course view, delete from DB
+    // Else the attainment is being edited during the creation of an instance so only delete from the context
+    if (instanceId) {
```

Comment:
Should this be inside try/catch? Or is error handling done somewhere else?

Reply:
Added try/catch